Automattic/sensei
# Add - Learning Mode template settings UI

Fixes #5539 

### Changes proposed in this Pull Request

* Updates the Learning Mode templates settings UI to the latest designs.
* Adds ability to preview the template's image screenshot before activating.
* Supports keyboard navigation!

### Testing instructions


* Go to Sensei > Settings > Appearance
* Preview any LM template by clicking on the template thumbnail
* Confirm that all the action buttons (Activate, Customize, Upgrade to Pro) works as expected. ⚠️ Note: ⚠️ This is a static implementation of the **Activate** action. It reloads the page instead of updating it in the background via AJAX. 


### Screenshot / Video


https://user-images.githubusercontent.com/2578542/189303240-bced81a5-8ec1-407e-8cf9-d0900c5af84c.mp4
